lib/driver.js

```diff
@@ -296,6 +297,14 @@ class XCUITestDriver extends BaseDriver {
     log.info(`Setting up ${this.isRealDevice() ? 'real device' : 'simulator'}`);
 
     if (this.isSimulator()) {
+      if (this.opts.shutdownOtherSimulators) {
+        if (!this.relaxedSecurityEnabled) {
```

Comment:
Why this restriction?

Reply:
I assume this operation as potentially unsecure, since it changes the state of devices, whcih are not controlled by the particular sessions. So, in case we have multiple parallel sessions running on a single machine, this capability might unexpectedly close simulators used by other sessions, making them fail.

Reply:
That is a valid concern, yes.

I think we need _comprehensive_ documentation on what the `relaxedSecurityEnabled` server flag allows. The more stuff put behind the flag, the less useful it is, since the higher chance there is that there is something allowed that is undesired.

Reply:
agree with isaac on this